src/Applications/LDAS_App/CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ ecbuild_add_executable (
    TARGET tile_bin2nc4.x 
    SOURCES tile_bin2nc4.F90
    LIBS MAPL)
+ 
+ecbuild_add_executable (
+   TARGET ascat_mask_maker.x
+   SOURCES util/inputs/ASCAT_sm_mask/ascat_mask_maker.F90
+   LIBS MAPL
+)   
```

Comment:
@weiyuan-jiang : Should we have a CMakeLists.txt file in the sub-dir?  Or is it ok to have only the CMakeLists.txt in ./LDAS_App ?  Also, can you please make sure that the new executable is built but **not** installed?  Thanks!

Reply:
Here I followed the example for mwrtm_bin2nc4.x on the next few lines which is:
`ecbuild_add_executable (
   TARGET mwrtm_bin2nc4.x
   SOURCES util/inputs/mwRTM_params/mwrtm_bin2nc4.F90
   LIBS GEOSlandassim_GridComp)`

Reply:
> @weiyuan-jiang : Any thoughts on the above? I'd like to merge this PR soon.
Yes, we can make it build but not install
If there is no other fortran files, we dont need to add all the CMakeLists.txt